modules/_labs/select/react/lib/Select.tsx

```diff
@@ -418,16 +404,16 @@ class Select extends React.Component<SelectProps, SelectState> {
     this.fireChangeEvent(this.normalizedOptions[index].value);
   };
 
-  handleMenuBlur = (event: React.FocusEvent): void => {
-    this.toggleMenu(false);
-  };
```

Comment:
With `preventDefault` removed on the mousedown handler on the Select Button, we can no longer hide the Menu on blur (I was hiding the Menu on blur to handle its click-outside-to-dismiss behavior). Hiding the Menu on blur prevents you from clicking on the Select Button to dismiss the Menu while its open given the way the event handlers interact.

Clicking the Button:
1. Gives focus to the Button, which triggers the Menu blur handler, which causes the Menu to start closing... 
2. And triggers the Button's click handler which sees the Menu closing and immediately re-opens it again (this is intended behavior: if you activate the Menu while it's closing, it should re-open).

Fortunately, I was able to use Popup's `useCloseOnOutsideClick` hook as a replacement (see below).